### PR TITLE
feature/#17-variables-and-global-scss-files

### LINKED ISSUE #17: Create _variables.scss and _global.scss
- Create _variables.scss and _global.scss to add styles that could be used in all components and pages.
- Migrate styles from App.vue file to corresponding files like `.button` and `.container`.

### PR DESCRIPTION
**Pull request by jsebax**
Create variables and global scss files

Closes #17

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -9,58 +9,22 @@
...
+import Navbar from "@/components/Navbar.vue";
+import Footer from "@/components/Footer.vue";
 
 export default {
   components: {
     Navbar,
-    Footer,
-  },
+    Footer
+  }
```
Comment (camigomez35) [inline src/App.vue]: Please user `npm run lint --fix`.

File: src/sass/_global.scss [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+	cursor: pointer;
+	text-decoration: none;
+	color: white;
+	font-weight: 600;
+	transition: background 0.5s;
+
+	&:hover {
+		background: #aa0000;
+	}
+}
```
Comment (camigomez35) [inline src/sass/_global.scss]: Don't use tabs in this file. Please, give it format.

File: src/App.vue (hunk 1 above)
Comment (jsebax) [reply src/App.vue]: Done

File: src/sass/_global.scss (hunk 2 above)
Comment (jsebax) [reply src/sass/_global.scss]: Done

**Pull request merged**